.github/workflows/build_platformio.yaml

```diff
@@ -44,8 +45,7 @@ jobs:
         with:
           path: ~/.cache/pip
           key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
+          restore-keys: ${{ runner.os }}-pip-
```

Comment:
This can be simplified to just
```yml
          key: ${{ runner.os }}-pip-pio
```
Because there are no relevant requirements.txt files and the cache is specific to platformio runs.